uiowa/uiowa01
# Feature: Collegiate theme updates

Initial work on issues #201, #200, #156.

# How to test

#201 

1. Run collegiate_basic install.
2. Create a new Basic Page.
3. Edit layout to see if Page Title and Breadcrumbs are listed in a separate section from the Body.

#200 

1. Run collegiate_basic install.
2. Create a new Basic Page.
3. Add new section layout.
4. Configure the section using the "Full-Width Container" and "Edge-to-Edge" options separately to see if default spacing on left and right is added for "Full-Width Container" option and see if default spacing is removed when "Edge-to-Edge" is selected.
5. Change the theme settings from the default "container" option to "full-width" and verify that default spacing on the left and right is added by default for each section.    Try selecting the "Edge-to-Edge" option to override the default spacing.
6.  Test how the page content area responds to smaller browser sizes.  You should test this on the Basic Page and on a page that doesn't have Layout Builder enabled like the Article content type. 

#156 

1.   Create a three-column layout and verify that the center column has a left and right margin of .625rem.
